uint8 mission_type
uint8 last_mission_type
uint8 is_broken
uint8 error_code
